docs/GettingStartedDocs/Contributors/ExperimentalSupportRHEL8.md

```diff
@@ -0,0 +1,99 @@
+# Experimental - Getting Started with Open Enclave on RHEL8
+
+## Platform requirements
+
+- RHEL8
+- A system which is SGX1 or SGX1+FLC capable. Please refer to [SGXSupportLevel.md](/docs/GettingStartedDocs/SGXSupportLevel.md) to determine the SGX support level for your target system.
+
+Note: Attestation is not supported on RHEL8.
+
+## Clone Open Enclave SDK repo from GitHub
+
+Use the following command to download the source code (make sure `git` is installed before doing this):
+
+```bash
+git clone https://github.com/openenclave/openenclave.git
+```
+
+This creates a source tree under the directory called openenclave.
+
+## Install project requirements
+
+First, change directory into the openenclave repository:
+
+```bash
+cd openenclave
+```
+
+Run the command below to install prerequisites.
+
+```bash
+ansible-playbook scripts/ansible/oe-contributors-setup-sgx1.yml
```

Comment:
Ansible needs to be installed first. How can ansible be installed on RHEL?

Reply:
ansible is developed by redhart. ansible comes pre-installed on all RHEL systems.

Reply:
"sudo scripts/ansible/install-ansible.sh" works on RHEL.

Reply:
I have added that the the docs. Thanks @nonpolarity